lte/gateway/deploy/roles/magma/files/patches/aioeventlet_fd_exception.patch

```diff
@@ -0,0 +1,14 @@
+--- a/aioeventlet.py
++++ b/aioeventlet.py
+@@ -159,7 +159,10 @@ class _Selector(asyncio.selectors._BaseSelectorImpl):
+         self._notified = {}
+         ready = []
+         for fd, events in notified.items():
+-            key = self.get_key(fd)
++            try:
++                key = self.get_key(fd)
++            except KeyError:
++                continue
```

Comment:
not quite sure if I should be unregistering the FD here... Will add a log here and hopefully reproduce somehow